lib/webdrivers/chromedriver.rb

```diff
@@ -73,18 +80,56 @@ def chrome_version
       end
 
       def chrome_on_windows
-        reg = 'HKLM:\SOFTWARE\Microsoft\Windows\CurrentVersion\App Paths\chrome.exe'
-        ps  = "(Get-Item (Get-ItemProperty '#{reg}').'(Default)').VersionInfo.ProductVersion"
-        `powershell #{ps}`
+        if browser_defined?
+          Webdrivers.logger.debug "Browser executable: '#{browser_exe}'"
+          return `powershell (Get-ItemProperty '#{browser_exe}').VersionInfo.ProductVersion`
+        end
+
+        # Default to Google Chrome
+        reg        = 'HKLM:\SOFTWARE\Microsoft\Windows\CurrentVersion\App Paths\chrome.exe'
```

Comment:
It looks like this is how chromedriver is finding the values: https://github.com/SeleniumHQ/selenium/wiki/ChromeDriver#requirements
I'm not sure if the above is different, but it might be different depending on the version of Windows.

Reply:
Yea, it's different. I tested the registry and it works on Windows 7, 10, Windows Server 2008, and 2016. Not sure if we want to support XP and 7 still.

Reply:
Yeah, windows 7 is not EOL, yet.

Reply:
Yeah, I meant to say XP only. Registry works on Windows 7.